### ISSUE TITLE
How to run two different jit models in two GPUs respectively in one scrip?

### ISSUE THREAD

**Issue by lzj9072** [pytorch/pytorch#29190]
I have an encoder-decoder model.  After converted encoder and decoder model into jit models, I want to load encoder on GPU:0 and the encoder outputs **Keys** and **Value**. Then I move the **Keys** and **Values** to GPU:1 since the decoder is loaded on GPU:1. 
    encoder = torch.jit.load(feat_model).cuda(0)
    gru_decoder = torch.jit.load(gru_model, map_location=torch.device("cpu")).cuda(1)

    loader = getLoader(data_path, batch_size)
    for data in loader:
        audio, label = data
        batch_size = audio.size(0)
        k, v = encoder(audio.type("torch.FloatTensor").cuda(0))
        k = k.cuda(1)
        v = v.cuda(1)
        hidden = torch.zeros(1, batch_size, 512).type("torch.FloatTensor").cuda(1)
        target = torch.tensor(sos_id).repeat(batch_size).cuda(1)

        for step in range(k.size(1)):
            probs, hidden = gru_decoder(target, hidden, k, v)
            target = torch.argmax(probs, dim=-1)
I have checked that target, hidden, k, v are on GPU:1. However, an error occur:
![image](https://user-images.githubusercontent.com/26057369/68198427-f640f280-fff6-11e9-822c-afc721ef838e.png)
It seems that the embedding layer in decoder is on GPU:0. But I have already set decoder on CUDA:1. 
Does anyone have any solutions or ideas? Thanks a lot.

**Comment by liyuanyaun** [comment]
did you solve this problem? could you share the solution? thank you very much

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
